src/Components/benchmarkapps/Wasm.Performance/Driver/Wasm.Performance.Driver.csproj

```diff
@@ -6,6 +6,8 @@
 
     <HasReferenceAssembly>false</HasReferenceAssembly>
     <IsProjectReferenceProvider>false</IsProjectReferenceProvider>
+    <IsShippingPackage>false</IsShippingPackage>
```

Comment:
@dotnet/aspnet-build does this look correct?

Reply:
@wtgodbe has confirmed (during an offline conversation) that this is the right fix: `<IsShippingPackage>false</IsShippingPackage>`